.travis.yml

```diff
@@ -3,7 +3,7 @@ language: ruby
 before_install:
   - gem install bundler -v '< 2'
 
-dist: trusty
+dist: xenial
```

Comment:
You may want to merge these changes into master in case we have to revert or anything.